department-of-veterans-affairs/caseflow
# Attorney Check Out | Prevent submitting already-decided issues

### Description
In #5466, we excluded issues that have already been decided from the attorney's decision, hiding them in the Select Dispositions view. However, we would still include them in the payload sent to the server at the end of the checkout process, leading to an `IssueRepositoryError` ([1](https://dsva.slack.com/files/U51RH55BR/FAT9ZG3J4/error.png), [2](https://dsva.slack.com/archives/C6E41RE92/p1526994154000473))

### Acceptance Criteria 
- [ ] Issues with [dispositions A-X](https://github.com/department-of-veterans-affairs/caseflow/blob/master/constants/VACOLS_DISPOSITIONS_BY_ID.json) should be excluded from payload sent to server at the end of Attorney checkout flow

### Testing Plan
1. Re-seed FACOLS, go to `/queue` as `BVASCASPER1`
2. Start Draft Decision checkout for Grover Kerluke
3. Confirm only 6 issues appear in Select Dispositions view
4. Complete checkout, confirm no `Caseflow::Error::IssueRepositoryError` appears after submitting

![already-decided-issues](https://user-images.githubusercontent.com/8533004/40366180-85142190-5da4-11e8-870e-bc5d99b88deb.gif)
